docs/user.markdown

```diff
@@ -258,3 +258,19 @@ update of `alidist`, which users will have to be done separately.
 In case some yet to appear bug in alibuild will force us to rebuild a
 previously built area, this will be widely publicized and users will get a warning
 when running the command.
+
+## Fetching updates
+
+aliBuild tries to determine whether a package should be rebuilt or not. In case
+the package's version (`tag:`) is a _branch_ name, aliBuild needs to query the
+remote repository to see if the remote branch has changed.
+
+Since the vast majority of recipes do not use branches as versions (they use
+actual, fixed tags instead), aliBuild does not perform any remote repository
+query by default, and uses previously cached information. This results in
+reduced network activity and an improved startup time.
```

Comment:
Ok, but what happens if I add a new tag? Will that be transparently fetched?

Reply:
Yes because this logic is inside `build_template.sh`.